Leanplum/Common/LPConstants.h

```diff
@@ -13,6 +13,7 @@
 #define LEANPLUM_API_VERSION @"1.0.6"
 #define LEANPLUM_CLIENT @"ios"
 #define LEANPLUM_SUPPORTED_ENCODING @"gzip"
+#define LP_IV @"__l3anplum_iv__"
```

Comment:
do we need this to be so obfuscated?

Reply:
I think this should be kept the same as the old SDK in cases where something was encrypted using this